src/transformers/models/rembert/tokenization_rembert.py

```diff
@@ -111,6 +111,9 @@ def __init__(
         mask_token="[MASK]",
         **kwargs,
     ):
+        # Mask token behave like a normal word, i.e. include the space before it
+        mask_token = AddedToken("[MASK]", lstrip=True, rstrip=False, normalized=False)
```

Comment:
```suggestion
        mask_token = AddedToken(mask_token, lstrip=True, rstrip=False, normalized=False)
```

Reply:
That is not working, as for the initial  RembertTokenizer  instantiate is created from  line 45 of this file which by default makes the mask_token="[MASK]"  (a string) but the subsequent call happens from [this](https://github.com/huggingface/transformers/blob/main/tests/test_tokenization_common.py#L4083) line where mask_token is already AddedToken instance but with the lstrip = False (that will eventually fail the [test case](https://github.com/huggingface/transformers/blob/main/tests/test_tokenization_common.py#L4128) )

Reply:
you have to put 
```suggestion
        mask_token = AddedToken(mask_token, lstrip=True, rstrip=False, normalized=False) is isinstance(mask_token, str) else mask_token
```

Reply:
I can do that , but I can't change the value of lstrip, which is private to class and defaults to False

Reply:
ok, sorry for the confusion. I the issue is that the way the slow tokenizer was saved is such that the stripping for the token is wrong, then you can copy paste the test we are talking about, and make sure you pass the mask token in the test (to overwrtie this) or just push a Rembert tokenizer to the hub and use this one for the test! 🤗

Reply:
This is much better. I'll do the needful. Thanks